test/Sentry.AspNetCore.Tests/SentryTracingMiddlewareTests.cs

```diff
@@ -1,4 +1,4 @@
-﻿#if !NETCOREAPP2_1
+#if !NETCOREAPP2_1
```

Comment:
BOM?

Reply:
not sure what has changes, at least, vs code doesn't spot it https://github1s.com/getsentry/sentry-dotnet/pull/1143